calculator.rb

```diff
@@ -0,0 +1,55 @@
+# 1st method - evaluate_oper(operator): evaluate user input to see if it is one of the four avaliable operators (either symbol or word) for this calculator program. If yes, method will return the input; if no, method will display error message. Assumption: only lowercase input
+def evaluate_oper(operator)
+  if ["add", "+", "subtract", "-", "multiply", "*", "divide", "/"].include?(operator)
```

Comment:
Consider using a while loop to continually ask for user input until it is valid.